clio-server/src/main/scala/org/broadinstitute/clio/server/service/IndexService.scala

```diff
@@ -0,0 +1,80 @@
+package org.broadinstitute.clio.server.service
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+import io.circe.Json
+import org.broadinstitute.clio.server.dataaccess.elasticsearch._
+import org.broadinstitute.clio.transfer.model.TransferIndex
+import org.broadinstitute.clio.util.model.{DocumentStatus, UpsertId}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.reflect.ClassTag
+
+abstract class IndexService[TI <: TransferIndex, D <: ClioDocument: ElasticsearchIndex](
+  persistenceService: PersistenceService,
+  searchService: SearchService,
+  val transferIndex: TI,
+  elasticsearchIndex: ElasticsearchIndex[D]
```

Comment:
Could we make this `implicit`? (I think) it should thread through nicely if we do.

Reply:
These three implicifications result in ambiguous implicit value errors... but it actually looks like we can just strip this parameter out entirely. And in PersistenceService and SearchService as well